Azure/azure-cli
# [Redis] `az redis import/export`: Add `--storage-subscription-id` support while importing/exporting data

**Related command**

**az redis**

**Description**
PR is to update azure cli management client with new api v2023-04-01. Following parameters are added:
- New `storage-subscription-id` parameter added while performing import data to the cache.
- New `storage-subscription-id` parameter added while performing exporting data from the cache.
Swagger changes can be found here : 
https://github.com/Azure/azure-rest-api-specs/pull/23726

**Testing Guide**
Once Managed Identity is enabled on the cache & appropriate access roles specified in storage account then below commands can be run to export/import data on a storage account in a different subscription: https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-managed-identity
`az redis export -n \'{name}\' -g \'{rg}\' --prefix \'{prefix}\' --container \'{containersasURL}\' --preferred-data-archive-auth-method ManagedIdentity --storage-subscription-id \'{storageSubscriptionId}\' : optional parameter to provide subscription id where the storage account resides. 
`az redis import -n \'{name}\' -g \'{rg}\' --files \'{filesasURL}\' --preferred-data-archive-auth-method ManagedIdentity --storage-subscription-id \'{storageSubscriptionId}\' : optional parameter to provide subscription id where the storage account resides. 



---

This checklist is used to make sure that common guidelines for a pull request are followed.
